Document deleting SCAP scan results (bsc#1262471) (#4937)
uyuni-project/uyuni-docs: en/modules/administration/pages/openscap.adoc

You can also use the {productname} API to view results, with the ``system.scap`` handler.

[[removing-scap-scan-results]]
== Removing SCAP Scan Results

Only SCAP scans that have passed the configured retention period can be deleted.

> You can configure the retention period in the Organization Configuration dialog.

Reviewer: ```suggestion
in the [literal]``Organization Configuration`` field.
```

Author: ```suggestion
You can configure the retention period in the [literal]``Organization Configuration`` section.
```

> .Procedure: Configuring the SCAP Results Retention Period

Reviewer: ```suggestion
Procedure: Configuring the SCAP results retention period
```

Author: ```suggestion
.Procedure: Configuring the SCAP results retention period
```

[role=procedure]
_____

. Navigate to menu:Home[My Organization > Configuration].

. Under [guimenu]``SCAP``, check [guimenu]``Allow Deletion of SCAP Results``.

. Enter a number of days in [guimenu]``Allow Deletion After`` (default is 90).

_____

> .Procedure: Deleting SCAP Scan Results

Reviewer: Same as above, please use sentence-case titles except for the page title.

Author: ```suggestion
.Procedure: Deleting SCAP scan results
```
